pokemongo_bot/event_handlers/telegram_handler.py

```diff
@@ -124,34 +164,57 @@ class TelegramHandler(EventHandler):
     def __init__(self, bot, config):
         self.bot = bot
         self.tbot = None
-        self.master = config.get('master', None)
+        master = config.get('master', None)
         self.pokemons = config.get('alert_catch', {})
         self.whoami = "TelegramHandler"
         self.config = config
+        if master == None:
+            self.master = None
+            return
```

Comment:
Yes. Actually, did not want to deal with having a "None" in the select statement (lines 181, 182).